persistence/sql/persister_verification.go

```diff
@@ -132,3 +136,128 @@ func (p *Persister) DeleteExpiredVerificationFlows(ctx context.Context, expiresA
 	}
 	return nil
 }
+func (p *Persister) UseVerificationCode(ctx context.Context, fID uuid.UUID, codeVal string) (*code.VerificationCode, error) {
+	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.UseVerificationCode")
+	defer span.End()
+
+	var verificationCode *code.VerificationCode
+
+	nid := p.NetworkID(ctx)
+
+	flowTableName := new(verification.Flow).TableName(ctx)
+
+	if err := sqlcon.HandleError(p.Transaction(ctx, func(ctx context.Context, tx *pop.Connection) (err error) {
+
+		/* #nosec G201 TableName is static */
+		if err := sqlcon.HandleError(tx.RawQuery(fmt.Sprintf("UPDATE %s SET submit_count = submit_count + 1 WHERE id = ? AND nid = ?", flowTableName), fID, nid).Exec()); err != nil {
```

Comment:
Nit, but here breaking the long lines would help readability as well IMHO ;).

Reply:
Good point. Also looking much better now. Done.